_posts/2021-03-29-Arts and Sciences Recipe Page.md

```diff
@@ -0,0 +1,145 @@
+    * \<tl> *tool* \</tl> to indicate a tool used in the recipe
+  * document specific tags:
+    * \<recNum> *recipeNum* \<recNum> to indicate that a number was the number of the recipe in the list (these numbers were marked on the document by the author)
+    * \<pageNum> *pageNum* \<pageNum> to indicate a physically marked page number. Note that these numbers don't line up with the page in the online manuscript due to blank pages and front matter at the beginning. For that reason and just to let researchers know that the author did in fact put page numbers, I felt that putting this tag in addition to the page tag was worth it.
+
+### What textual structure seems appropriate? Consider the hierarchy of your tags.
+
+On a broad organizational level, the document was separated as a recipe list, with a number, title, and (sometimes) multiple paragraphs of writing in each recipe. In addition, there were page numbers and breaks that frequently appeared in the middle of recipes (but not paragraphs, since that doesn't really make sense).
+
+To account for this organization, I broadly formatted my text with paragraph tags, page tags, recipe number tags, and heading tags nested directly inside div tags (with type = "recipe"). Margin text occasionally appeared alongside title content outside of paragraphs, so there are margin tags nested directly inside div tags as well. Also, some lines or double lines appear between titles and paragraphs, so these tags are also nested directly inside div tags. (double lines also appear outside of recipes, so these are the only tag put outside of div tags).
+
+In terms of finer textual structure, there was some tension about  hierarchy when physical aspects of the texts. For example, there were chunks of text including headings/titles, recipe numbers, lines, and margin writing that were all a different ink color (red or brown instead of black). In these cases, I chose to put color tags as the children of other writing or physical tags (aside from the line or double line tags since these are marks themselves rather than indicators of a physical property of enclosed writing).
+
+The finest textual elements were writing aspects which (aside from the 2 document-specific tags) could be reasonably marked as the child of any non-writing aspect tag since they were small in size and clearly contained within other tags. For example, material tags such as that for Oil of Turpentine clearly should be the child of broader tags such as those for paragraphs. Hierarchy between these writing tags was also easily decided by size; for Neopolitan Ointment, for example, the Neopolitan place descriptor is contained within the material tag for Neopolitan Ointment as a whole.  
+
+### What questions about markup emerged throughout the process?
+
+I primarily ran into a lot of hierarchy and how-much-to-tag, but these aren't really questions about markup so much as personal preference and typical practice.
```

Comment:
Yes, and if this were a real XML project, these would be questions that you as team lead would decide upon at the start of the project. You'd create a schema of tags to work from, and then your team would follow your decisions. Sometimes it really does come down to the personal preference of the team lead, but the TEI standards are also useful as a way of maintaining consistency across projects.